packages/commons-test/src/riskAnalysisTestUtils.ts

```diff
@@ -42,6 +67,35 @@ export const validRiskAnalysis3_0_Pa: RiskAnalysisFormToValidate = {
   },
 };
 
+export const validatedRiskAnalysis2_0_Pa: RiskAnalysisValidatedForm = {
```

Comment:
Maybe also here we could explicit that this is actually expired
```suggestion
export const validatedRiskAnalysis2_0_Pa_Expired: RiskAnalysisValidatedForm = {
```